Makefile.am

```diff
@@ -132,6 +132,8 @@ linux_platform_headers = \
 
 if HTOP_LINUX
 AM_LDFLAGS += -rdynamic
+myhtopsources += SELinuxMeter.c
+myhtopheaders += SELinuxMeter.h
```

Comment:
Should these new sources reside below the linux/ directory?  I guess SE*Linux* isn't going to exist on other platforms ;)
Otherwise LGTM.

Reply:
moved into the linux/ directory.